artemis-protocols/artemis-openwire-protocol/src/main/java/org/apache/activemq/artemis/core/protocol/openwire/OpenWireProtocolManager.java

```diff
@@ -84,6 +85,10 @@
 import org.apache.activemq.util.LongSequenceGenerator;
 
 public class OpenWireProtocolManager implements ProtocolManager<Interceptor>, ClusterTopologyListener {
+   static {
+      // this is not really a property, needs to be ignored
+      FluentPropertyBeanIntrospectorWithIgnores.addIgnore(OpenWireProtocolManager.class.getName(), "setUpInactivityParams");
```

Comment:
Instead of that.. what about renaming the method to configureInactivityParams.

You wouldn't need to do anything with Beans.